applications/RANSApplication/python_scripts/formulations/turbulence_models/scalar_turbulence_model_rans_formulation.py

```diff
@@ -0,0 +1,208 @@
+            self.GetModelPart(),
+            scheme,
+            convergence_criteria,
+            builder_and_solver,
+            settings["max_iterations"].GetInt(),
+            False,
+            False,
+            False)
+
+        self.solver.SetEchoLevel(self.echo_level)
+        convergence_criteria.SetEchoLevel(self.echo_level)
+
+        super().Initialize()
+        Kratos.Logger.PrintInfo(self.__class__.__name__, "Initialized formulation")
+
+    def SolveCouplingStep(self):
+        if (self.IsBufferInitialized()):
+            self.ExecuteBeforeCouplingSolveStep()
+            self.solver.Predict()
+            self.solver.SolveSolutionStep()
+            self.ExecuteAfterCouplingSolveStep()
+            Kratos.Logger.PrintInfo(self.__class__.__name__, "Solved  formulation.")
+            return True
+
+        return False
+
+    def GetStrategy(self):
+        return self.solver
+
+    def SetTimeSchemeSettings(self, settings):
+        if (settings.Has("scheme_type")):
+            scheme_type = settings["scheme_type"].GetString()
+            if (scheme_type == "steady"):
+                self.is_steady_simulation = True
+            elif (scheme_type == "bdf2" or scheme_type == "bossak"):
+                self.is_steady_simulation = False
+            else:
+                raise Exception(
+                    "Only \"steady\", \"bdf2\" and \"bossak\" scheme types supported. [ scheme_type = \""
+                    + scheme_type + "\" ]")
+        else:
+            raise Exception(
+                "\"scheme_type\" is missing in time scheme settings")
+
+    def GetMaxCouplingIterations(self):
+        return "N/A"
+
+    def GetModelPart(self):
+        return self.turbulence_model_part
+
+    def SetStabilizationMethod(self, stabilization_method):
+        self.element_name = self.GetElementNamePrefix()
+        if (stabilization_method == "algebraic_flux_corrected"):
+            self.element_name = self.element_name + "AFC"
+            self.scheme_type = self._CreateAlgebraicFluxCorrectedSteadyScalarScheme
+        elif (stabilization_method == "residual_based_flux_corrected"):
+            self.element_name = self.element_name + "RFC"
+            self.scheme_type = GetKratosObjectType("SteadyScalarScheme")
+        elif (stabilization_method == "non_linear_cross_wind_dissipation"):
+            self.element_name = self.element_name + "CWD"
+            self.scheme_type = GetKratosObjectType("SteadyScalarScheme")
+        else:
+            raise Exception("Unsupported stabilization method")
+
+    def SetWallFunctionSettings(self, settings):
+        self.condition_name = self.GetConditionNamePrefix()
+
+        if (self.condition_name != ""):
+            wall_function_region_type = "logarithmic_region_only"
+            if (settings.Has("wall_function_region_type")):
+                wall_function_region_type = settings[
+                    "wall_function_region_type"].GetString()
```

Comment:
```suggestion
            if (settings.Has("wall_function_region_type")):
                wall_function_region_type = settings["wall_function_region_type"].GetString()
            else:
                wall_function_region_type = "logarithmic_region_only"         
```
I think it's more clear in this way.

Reply:
done :)